src/main/java/com/intellias/basicsandbox/config/security/basic/BasicAuthSecurityConfig.java

```diff
@@ -0,0 +1,54 @@
+package com.intellias.basicsandbox.config.security.basic;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.PropertySource;
+import org.springframework.security.config.annotation.method.configuration.EnableMethodSecurity;
+import org.springframework.security.config.annotation.web.builders.HttpSecurity;
+import org.springframework.security.core.userdetails.User;
+import org.springframework.security.core.userdetails.UserDetails;
+import org.springframework.security.crypto.bcrypt.BCryptPasswordEncoder;
+import org.springframework.security.crypto.password.PasswordEncoder;
+import org.springframework.security.provisioning.InMemoryUserDetailsManager;
+import org.springframework.security.web.SecurityFilterChain;
+
+@Configuration
+@EnableMethodSecurity
+@PropertySource("classpath:credentials.properties")
```

Comment:
probably to use yml instead of properties? this format is being used across the entire project.

Reply:
Resolving this comment as after the discussion we decided to leave it as is.